install-modules: seems not work on Windows

```
            - uses: shogo82148/actions-setup-perl@v1
              with:
                  install-modules: Test2::V0
            - run: perl -MTest2::V0 -e '#'
```
Fails:
```
2021-12-07T08:19:21.5823710Z Download action repository 'shogo82148/actions-setup-perl@v1' (SHA:88d7620362b0c09f8cf08c22d3baf4ce914a70ff)
...
2021-12-07T08:19:24.8098677Z ##[group]Run shogo82148/actions-setup-perl@v1
2021-12-07T08:19:24.8099692Z with:
2021-12-07T08:19:24.8100231Z   install-modules: Test2::V0
2021-12-07T08:19:24.8101112Z   perl-version: 5
2021-12-07T08:19:24.8102121Z   distribution: default
2021-12-07T08:19:24.8102725Z   enable-modules-cache: true
2021-12-07T08:19:24.8103323Z   working-directory: .
2021-12-07T08:19:24.8103863Z ##[endgroup]
2021-12-07T08:19:26.8743365Z ##[group]install perl
2021-12-07T08:19:29.6216322Z [command]"C:\Program Files\PowerShell\7\pwsh.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$ErrorActionPreference = 'Stop' ; try { Add-Type -AssemblyName System.IO.Compression.ZipFile } catch { } ; try { [System.IO.Compression.ZipFile]::ExtractToDirectory('D:\a\_temp\bd85cb30-2474-42e9-bfdc-7fc835c094f8', 'D:\a\_temp\c302fa82-f545-4d71-9f6a-247a2d7c8053', $true) } catch { if (($_.Exception.GetType().FullName -eq 'System.Management.Automation.MethodException') -or ($_.Exception.GetType().FullName -eq 'System.Management.Automation.RuntimeException') ){ Expand-Archive -LiteralPath 'D:\a\_temp\bd85cb30-2474-42e9-bfdc-7fc835c094f8' -DestinationPath 'D:\a\_temp\c302fa82-f545-4d71-9f6a-247a2d7c8053' -Force } else { throw $_ } } ;"
2021-12-07T08:19:36.1966397Z 
2021-12-07T08:19:36.2333376Z [command]C:\hostedtoolcache\windows\perl\5.34.0-thr\x64\bin\perl.exe -V
2021-12-07T08:19:36.7994027Z Summary of my perl5 (revision 5 version 34 subversion 0) configuration:
2021-12-07T08:19:36.7996210Z    
2021-12-07T08:19:36.7996920Z   Platform:
2021-12-07T08:19:36.7997919Z     osname=MSWin32
2021-12-07T08:19:36.7998453Z     osvers=10.0.17763.2300
2021-12-07T08:19:36.8000241Z     archname=MSWin32-x64-multi-thread
2021-12-07T08:19:36.8006389Z     uname=''
2021-12-07T08:19:36.8011085Z     config_args='undef'
2021-12-07T08:19:36.8011879Z     hint=recommended
2021-12-07T08:19:36.8012476Z     useposix=true
2021-12-07T08:19:36.8013040Z     d_sigaction=undef
2021-12-07T08:19:36.8013661Z     useithreads=define
2021-12-07T08:19:36.8014328Z     usemultiplicity=define
2021-12-07T08:19:36.8014955Z     use64bitint=define
2021-12-07T08:19:36.8015546Z     use64bitall=undef
2021-12-07T08:19:36.8016151Z     uselongdouble=undef
2021-12-07T08:19:36.8016739Z     usemymalloc=n
2021-12-07T08:19:36.8017691Z     default_inc_excludes_dot=define
2021-12-07T08:19:36.8018335Z   Compiler:
2021-12-07T08:19:36.8018854Z     cc='gcc'
2021-12-07T08:19:36.8020153Z     ccflags =' -DWIN32 -DWIN64  -DPERL_TEXTMODE_SCRIPTS -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DUSE_PERLIO -D__USE_MINGW_ANSI_STDIO -fwrapv -fno-strict-aliasing -mms-bitfields'
2021-12-07T08:19:36.8021741Z     optimize='-s -O2'
2021-12-07T08:19:36.8022338Z     cppflags='-DWIN32'
2021-12-07T08:19:36.8022902Z     ccversion=''
2021-12-07T08:19:36.8023471Z     gccversion='8.1.0'
2021-12-07T08:19:36.8024043Z     gccosandvers=''
2021-12-07T08:19:36.8024595Z     intsize=4
2021-12-07T08:19:36.8025109Z     longsize=4
2021-12-07T08:19:36.8025650Z     ptrsize=8
2021-12-07T08:19:36.8026185Z     doublesize=8
2021-12-07T08:19:36.8026742Z     byteorder=12345678
2021-12-07T08:19:36.8027334Z     doublekind=3
2021-12-07T08:19:36.8027889Z     d_longlong=define
2021-12-07T08:19:36.8028459Z     longlongsize=8
2021-12-07T08:19:36.8029008Z     d_longdbl=define
2021-12-07T08:19:36.8030340Z     longdblsize=16
2021-12-07T08:19:36.8031071Z     longdblkind=3
2021-12-07T08:19:36.8031631Z     ivtype='long long'
2021-12-07T08:19:36.8032160Z     ivsize=8
2021-12-07T08:19:36.8033037Z     nvtype='double'
2021-12-07T08:19:36.8033543Z     nvsize=8
2021-12-07T08:19:36.8034038Z     Off_t='long long'
2021-12-07T08:19:36.8034550Z     lseeksize=8
2021-12-07T08:19:36.8035088Z     alignbytes=8
2021-12-07T08:19:36.8035620Z     prototype=define
2021-12-07T08:19:36.8036225Z   Linker and Libraries:
2021-12-07T08:19:36.8036795Z     ld='g++'
2021-12-07T08:19:36.8040587Z     ldflags ='-s -L"C:\hostedtoolcache\windows\perl\5.34.0-thr\x64\lib\CORE" -L"C:\strawberry\c\lib" -L"C:\strawberry\c\x86_64-w64-mingw32\lib" -L"C:\strawberry\c\lib\gcc\x86_64-w64-mingw32\8.1.0"'
2021-12-07T08:19:36.8042638Z     libpth=C:\strawberry\c\lib C:\strawberry\c\x86_64-w64-mingw32\lib C:\strawberry\c\lib\gcc\x86_64-w64-mingw32\8.1.0
2021-12-07T08:19:36.8044238Z     libs= -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lws2_32 -lmpr -lwinmm -lversion -lodbc32 -lodbccp32 -lcomctl32
2021-12-07T08:19:36.8046107Z     perllibs= -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lws2_32 -lmpr -lwinmm -lversion -lodbc32 -lodbccp32 -lcomctl32
2021-12-07T08:19:36.8047248Z     libc=
2021-12-07T08:19:36.8047715Z     so=dll
2021-12-07T08:19:36.8048227Z     useshrplib=true
2021-12-07T08:19:36.8048823Z     libperl=libperl534.a
2021-12-07T08:19:36.8049617Z     gnulibc_version=''
2021-12-07T08:19:36.8050338Z   Dynamic Linking:
2021-12-07T08:19:36.8050922Z     dlsrc=dl_win32.xs
2021-12-07T08:19:36.8051461Z     dlext=dll
2021-12-07T08:19:36.8051976Z     d_dlsymun=undef
2021-12-07T08:19:36.8052671Z     ccdlflags=' '
2021-12-07T08:19:36.8053182Z     cccdlflags=' '
2021-12-07T08:19:36.8054381Z     lddlflags='-mdll -s -L"C:\hostedtoolcache\windows\perl\5.34.0-thr\x64\lib\CORE" -L"C:\strawberry\c\lib" -L"C:\strawberry\c\x86_64-w64-mingw32\lib" -L"C:\strawberry\c\lib\gcc\x86_64-w64-mingw32\8.1.0"'
2021-12-07T08:19:36.8055382Z 
2021-12-07T08:19:36.8055727Z 
2021-12-07T08:19:36.8056708Z Characteristics of this binary (from libperl): 
2021-12-07T08:19:36.8057428Z   Compile-time options:
2021-12-07T08:19:36.8057996Z     HAS_TIMES
2021-12-07T08:19:36.8058517Z     HAVE_INTERP_INTERN
2021-12-07T08:19:36.8059050Z     MULTIPLICITY
2021-12-07T08:19:36.8059765Z     PERLIO_LAYERS
2021-12-07T08:19:36.8061201Z     PERL_COPY_ON_WRITE
2021-12-07T08:19:36.8061780Z     PERL_DONT_CREATE_GVSV
2021-12-07T08:19:36.8062573Z     PERL_IMPLICIT_CONTEXT
2021-12-07T08:19:36.8063143Z     PERL_IMPLICIT_SYS
2021-12-07T08:19:36.8063694Z     PERL_MALLOC_WRAP
2021-12-07T08:19:36.8064229Z     PERL_OP_PARENT
2021-12-07T08:19:36.8064819Z     PERL_PRESERVE_IVUV
2021-12-07T08:19:36.8065372Z     USE_64_BIT_INT
2021-12-07T08:19:36.8065883Z     USE_ITHREADS
2021-12-07T08:19:36.8066424Z     USE_LARGE_FILES
2021-12-07T08:19:36.8066952Z     USE_LOCALE
2021-12-07T08:19:36.8067556Z     USE_LOCALE_COLLATE
2021-12-07T08:19:36.8068085Z     USE_LOCALE_CTYPE
2021-12-07T08:19:36.8068629Z     USE_LOCALE_NUMERIC
2021-12-07T08:19:36.8069351Z     USE_LOCALE_TIME
2021-12-07T08:19:36.8070049Z     USE_PERLIO
2021-12-07T08:19:36.8070623Z     USE_PERL_ATOF
2021-12-07T08:19:36.8071317Z   Built under MSWin32
2021-12-07T08:19:36.8071892Z   Compiled at Nov 23 2021 07:13:21
2021-12-07T08:19:36.8072591Z   @INC:
2021-12-07T08:19:36.8073229Z     C:/hostedtoolcache/windows/perl/5.34.0-thr/x64/site/lib
2021-12-07T08:19:36.8074028Z     C:/hostedtoolcache/windows/perl/5.34.0-thr/x64/lib
2021-12-07T08:19:36.8074916Z 
2021-12-07T08:19:36.8198425Z ##[endgroup]
2021-12-07T08:19:36.8199500Z ##[group]install CPAN modules
2021-12-07T08:19:36.8200391Z nothing to install
2021-12-07T08:19:36.8201371Z ##[endgroup]
2021-12-07T08:19:39.5752902Z ##[group]Run perl -MTest2::V0 -e '#'
2021-12-07T08:19:39.5753867Z [36;1mperl -MTest2::V0 -e '#'[0m
2021-12-07T08:19:39.5826206Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2021-12-07T08:19:39.5826912Z env:
2021-12-07T08:19:39.5827614Z   PERL5LIB: D:\a\_actions\shogo82148\actions-setup-perl\v1\scripts\lib
2021-12-07T08:19:39.5828398Z ##[endgroup]
2021-12-07T08:19:39.9909176Z Can't locate Test2/V0.pm in @INC (you may need to install the Test2::V0 module) (@INC contains: D:\a\_actions\shogo82148\actions-setup-perl\v1\scripts\lib C:/hostedtoolcache/windows/perl/5.34.0-thr/x64/site/lib C:/hostedtoolcache/windows/perl/5.34.0-thr/x64/lib).
2021-12-07T08:19:39.9912095Z BEGIN failed--compilation aborted.
2021-12-07T08:19:40.3565102Z ##[error]Process completed with exit code 1.
2021-12-07T08:19:40.3742887Z Cleaning up orphan processes
```
But,
```
            - uses: shogo82148/actions-setup-perl@v1
            - run: cpanm --notest Test2::V0
            - run: perl -MTest2::V0 -e '#'
```
Works:
```
...
2021-12-07T08:28:28.8346074Z ##[group]Run cpanm --notest Test2::V0
2021-12-07T08:28:28.8346872Z [36;1mcpanm --notest Test2::V0[0m
2021-12-07T08:28:28.8406596Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2021-12-07T08:28:28.8407182Z env:
2021-12-07T08:28:28.8407848Z   PERL5LIB: D:\a\_actions\shogo82148\actions-setup-perl\v1\scripts\lib
2021-12-07T08:28:28.8408498Z ##[endgroup]
2021-12-07T08:28:30.2265398Z --> Working on Test2::V0
2021-12-07T08:28:30.7816814Z Fetching http://www.cpan.org/authors/id/E/EX/EXODIST/Test2-Suite-0.000144.tar.gz ... OK
2021-12-07T08:28:33.7490612Z Configuring Test2-Suite-0.000144 ... OK
2021-12-07T08:28:33.7523105Z ==> Found dependencies: Importer, Sub::Info, Module::Pluggable, Term::Table, Scope::Guard
2021-12-07T08:28:33.7528328Z --> Working on Importer
2021-12-07T08:28:33.7923749Z Fetching http://www.cpan.org/authors/id/E/EX/EXODIST/Importer-0.026.tar.gz ... OK
2021-12-07T08:28:34.3137209Z Configuring Importer-0.026 ... OK
2021-12-07T08:28:36.2489493Z Building Importer-0.026 ... OK
2021-12-07T08:28:36.2507796Z Successfully installed Importer-0.026
2021-12-07T08:28:36.3796846Z --> Working on Sub::Info
2021-12-07T08:28:36.4185058Z Fetching http://www.cpan.org/authors/id/E/EX/EXODIST/Sub-Info-0.002.tar.gz ... OK
2021-12-07T08:28:36.9723701Z Configuring Sub-Info-0.002 ... OK
2021-12-07T08:28:39.6514232Z Building Sub-Info-0.002 ... OK
2021-12-07T08:28:39.6532811Z Successfully installed Sub-Info-0.002
2021-12-07T08:28:39.7812372Z --> Working on Module::Pluggable
2021-12-07T08:28:39.8681434Z Fetching http://www.cpan.org/authors/id/S/SI/SIMONW/Module-Pluggable-5.2.tar.gz ... OK
2021-12-07T08:28:40.8812032Z Configuring Module-Pluggable-5.2 ... OK
2021-12-07T08:28:42.8144984Z Building Module-Pluggable-5.2 ... OK
2021-12-07T08:28:42.8164862Z Successfully installed Module-Pluggable-5.2
2021-12-07T08:28:42.9516523Z --> Working on Term::Table
2021-12-07T08:28:43.0432328Z Fetching http://www.cpan.org/authors/id/E/EX/EXODIST/Term-Table-0.015.tar.gz ... OK
2021-12-07T08:28:43.6342347Z Configuring Term-Table-0.015 ... OK
2021-12-07T08:28:45.7270333Z Building Term-Table-0.015 ... OK
2021-12-07T08:28:45.7277064Z Successfully installed Term-Table-0.015
2021-12-07T08:28:45.8614855Z --> Working on Scope::Guard
2021-12-07T08:28:45.9040312Z Fetching http://www.cpan.org/authors/id/C/CH/CHOCOLATE/Scope-Guard-0.21.tar.gz ... OK
2021-12-07T08:28:46.4566956Z Configuring Scope-Guard-0.21 ... OK
2021-12-07T08:28:48.3738761Z Building Scope-Guard-0.21 ... OK
2021-12-07T08:28:48.3740155Z Successfully installed Scope-Guard-0.21
2021-12-07T08:28:52.3690387Z Building Test2-Suite-0.000144 ... OK
2021-12-07T08:28:52.3704552Z Successfully installed Test2-Suite-0.000144
2021-12-07T08:28:52.9961918Z 6 distributions installed
2021-12-07T08:28:54.7173401Z ##[group]Run perl -MTest2::V0 -e '#'
2021-12-07T08:28:54.7174032Z [36;1mperl -MTest2::V0 -e '#'[0m
2021-12-07T08:28:54.7227118Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2021-12-07T08:28:54.7227717Z env:
2021-12-07T08:28:54.7228335Z   PERL5LIB: D:\a\_actions\shogo82148\actions-setup-perl\v1\scripts\lib
2021-12-07T08:28:54.7229132Z ##[endgroup]

```

--- a/src/cpan-installer.ts
+++ b/src/cpan-installer.ts
@@ -24,12 +24,8 @@ export interface Options {
 }
 
 export async function install(opt: Options): Promise<void> {
-  if (!opt.install_modules_with) {
-    core.info("nothing to install");
-    return;
-  }
   let installer: (opt: Options) => Promise<void>;
-  switch (opt.install_modules_with) {
+  switch (opt.install_modules_with || "cpanm") {
     case "cpanm":
       installer = installWithCpanm;
       break;
@@ -162,8 +158,10 @@ async function installWithCpanm(opt: Options): Promise<void> {
     args.push("--verbose");
   }
   args.push(...splitArgs(opt.install_modules_args));
-  if (await exists(path.join(workingDirectory, "cpanfile"))) {
-    await exec.exec(perl, [...args, "--installdeps", "."], execOpt);
+  if (opt.install_modules_with) {
+    if (await exists(path.join(workingDirectory, "cpanfile"))) {
+      await exec.exec(perl, [...args, "--installdeps", "."], execOpt);
+    }
   }
   const modules = splitModules(opt.install_modules);
   if (modules.length > 0) {

